src/features/admin/project-assigments/project-assignment-form/project-assignment-form.tsx

```diff
@@ -319,12 +320,13 @@ export const ProjectAssignmentForm = () => {
         abortEarly: false,
       })
       const existingProjectMemberIds = project_members.map((member) => member.id)
-      if (
-        project_members.length > 0 &&
-        !existingProjectMemberIds.includes(parseInt(projectMemberFormData?.user_id ?? ""))
-      ) {
-        setShowOverlapDialog(true)
-        return
+      const userId = projectMemberFormData?.user_id
+      if (userId !== undefined && !isNaN(parseInt(userId, 10))) {
+        const parsedUserId = parseInt(userId, 10)
```

Comment:
Define parsedUserId before the if statement, and use it like this `if (userId !== undefined && !isNaN(parsedUserId))`